stdlib/public/core/MutableCollection.swift

```diff
@@ -269,6 +284,45 @@ extension MutableCollection {
   }
 }
 
+@available(SwiftStdlib 5.5, *)
```

Comment:
IIUC, we would only land this on `main`, so if we needed to add APIs with availability, we wouldn't be able to use the 5.5 cohort here. (We need to add a placeholder marker for the "next" version, whatever its number may become. `SwiftStdlib 9999` seems appropriate.)

Reply:
I hadn't tried `SwiftStdlib 9999`, thank you. I tried the list of OSs at 9999, and that did *not* work.

Reply:
IIRC the compiler doesn't understand that 9999 is supposed to sometimes be available -- so it only works when there is an explicit runtime availability check. (Or within declarations that are also 9999-available.)